src/isomorphic/classic/element/ReactElementType.js

```diff
@@ -23,7 +23,7 @@ export type ReactElement = {
   $$typeof: any,
   type: any,
   key: any,
-  ref: any,
+  ref: string | (elem: ?HTMLElement) => void,
```

Comment:
There's already a command `flow coverage <file>` but it just shows the summary `Covered: 44.86% (48 of 107 expressions)`

Would be awesome if it printed in ascii art what nuclide shows:

<img width="747" alt="screen shot 2016-08-29 at 9 02 43 am" src="https://cloud.githubusercontent.com/assets/197597/18058061/84f0fe6e-6dc7-11e6-96f4-ca195c18d882.png">